src/quo/components/wallet/transaction_progress/view.cljs

```diff
@@ -128,24 +120,16 @@
      :collectible-number tag-number
      :type               :collectible}]])
 
-(defn- get-network-text
-  [network]
-  (case network
-    :arbitrum (i18n/label :t/arbitrum)
-    :mainnet  (i18n/label :t/mainnet)
-    :optimism (i18n/label :t/optimism)
-    nil))
-
```

Comment:
The changes in `quo/transaction-progress` are done to remove dependency on this function